LondonBritishLibrary/add/BLadd20741.xml

```diff
@@ -107,6 +135,245 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         </adminInfo>
                     </additional>
+                    </msPart>
+                    
+                    <msPart xml:id="p2">
+                        <msIdentifier>
+                            <idno>BL Additional 20741</idno>
+                            <altIdentifier>
+                                <idno>Wright cat. CLXX</idno>
+                            </altIdentifier>
+                            <altIdentifier>
+                                <idno>Wright 170</idno>
+                            </altIdentifier>
+                        </msIdentifier>
+                        <msContents>
+                            <summary/>
+                            
+                            <msItem xml:id="p2_i1">
+                                <locus target="#15r"/>
+                                <title type="incomplete" ref="LIT6677PPDeliverance"/>
+                                <textLang mainLang="gez"/>
+                                <note>Incomplete at the beginning.</note>
+                            </msItem>
+                            
+                            <msItem xml:id="p2_i2">
+                                <locus from="15r"/>
+                                <title type="complete" ref="LIT6678PPLightning">፺፩ ጸሎተ፡ መብረቅ፡</title>
+                                <textLang mainLang="gez"/>
+                            </msItem>
+                            
+                            <msItem xml:id="p2_i3">
+                                <locus target="#15v"/>
+                                <title type="complete" ref="LIT6679PPCaptivity">፺፪ ጸሎተ፡ ሞቅሕ፡</title>
+                                <textLang mainLang="gez"/>
+                            </msItem>
+                            
+                            <msItem xml:id="p2_i4">
+                                <locus target="#15v"/>
+                                <title type="complete" ref="LIT6680PPQwersat">፺፫፡ <gap reason="ellipsis"/> ሕማመ፡ ቍርፀት፡</title>
+                                <textLang mainLang="gez"/>
+                            </msItem>
+                            
+                            <msItem xml:id="p2_i5">
+                                <locus from="15v"/>
+                                <title type="complete" ref="LIT6681PPDemon"/>
+                                <textLang mainLang="gez"/>
+                            </msItem>
+                            
+                            <msItem xml:id="p2_i6">
+                                <locus target="#16r"/>
+                                <title type="complete" ref="LIT6682PPMichael">ጸሎተ፡ ሚካኤል፡</title>
+                                <textLang mainLang="gez"/>
+                            </msItem>
+                            
+                            <msItem xml:id="p2_i7">
+                                <locus target="#16r"/>
+                                <title type="complete"/>
```

Comment:
at least something to fill the title
e.g.
`<title type="complete">Prayer</title>`?